File: translation_output/mistral/avatar/python/java-sanitized/codeforces_514_A.java
import java.util.Scanner;

public class codeforces_514_A {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a number: ");
        String ip = sc.nextLine();
        String st = " ";
        if (ip.length() != 1) {
            if (ip.charAt(0) == '9') {
                st = "9";
                ip = ip.substring(1);
            }
            for (int i = 0; i < ip.length(); i++) {
                int num = Integer.parseInt(ip.charAt(i) + "");
                if (num > 4) {
                    int n = 9 - num;
                    st += Integer.toString(n);
                } else {
                    st += Integer.toString(num);
                }
            }
        } else {
            st = ip;
        }
        System.out.println(Integer.parseInt(st));
    }
}

File: translation_output/no_test/llama2/avatar/java/python-sanitized/codeforces_514_A.py
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

class codeforces_514_A:
    def main(self, args):
        sc = Scanner(System.in)
        list = []
        x = sc.nextLong()

        while x > 0:
            r = x % 10
            if 9 - r < r:
                if x / 10 == 0 and 9 - r == 0:
                    list.append(r)
                else:
                    list.append(9 - r)
            else:
                list.append(r)
            x = x / 10

        pow = 0
        newNumber = 0

        for i in range(len(list)):
            newNumber = newNumber + list[i] * (long)(math.pow(10, pow))
            pow += 1

        print(newNumber)